doc/src/guides/solving/solve-system-of-inequalities-algebraically.md

```diff
@@ -0,0 +1,218 @@
+# Reduce a System of Inequalities for a Single Variable Algebraically
+
+Use SymPy to reduce a system of inequalities for a single variable
+algebraically. For example, reducing $x^2 < \pi$, $x > 0$ yields $0 < x <
+\sqrt{\pi}$.
+
+```{note}
+SymPy can currently reduce for only one symbol (variable) in a system of 
+inequalities. SymPy can reduce a system containing more than one symbol.
+```
+
+Alternatives to consider:
+- To reduce for more than one symbol, try SciPy's
+  {external:func}`~scipy.optimize.linprog`
+- To reduce Boolean expressions, use {func}`as_set
+  <sympy.logic.boolalg.Boolean.as_set>`
+
+Here is an example of reducing a system of inequalities for a single variable
+algebraically. {func}`~.reduce_inequalities` accepts a list or tuple of
+inequalities to be reduced as a system:
+
+```py
+>>> from sympy import symbols, reduce_inequalities, pi
+>>> x = symbols('x')
+>>> reduce_inequalities([x >= 0, x**2 <= pi], x)
+(0 <= x) & (x <= sqrt(pi))
+```
+
+```{note}
+While {func}`~.solve` currently accomplishes the same thing (by calling 
+{func}`~.reduce_inequalities` internally), that functionality may be 
+deprecated or removed from {func}`~.solve`. We thus recommend using 
+{func}`~.reduce_inequalities`.
+```
+
+{func}`~.reduce_inequalities` is the top-level inequality-reducing function
+which will internally call any other lower-level [inequality-reducing
+functions](../../modules/solvers/inequalities.rst) as needed.
+
+## Guidance
+
+### Include the Variable to Be Reduced for in the Function Call
+
+We recommend you include the variable to be reduced for as the second argument
+for {func}`~.reduce_inequalities` to ensure that it reduces for the desired
+variable.
+
+### Recognize That SymPy Can Reduce for Only One Variable
+
+SymPy can currently reduce for only one symbol (variable) in a system of
+inequalities. SymPy can reduce a system containing more than one symbol. For
+example, the following system of inequalities has two variables, $x$ and $y$.
+SymPy can reduce for $x$, and gives the constraints on $y$.
+
+```py
+>>> from sympy import reduce_inequalities, symbols
+>>> x, y = symbols("x y")
+>>> reduce_inequalities([x + y > 1, y > 0], x)
+(0 < y) & (y < oo) & (x > 1 - y)
+```
+
+However, SymPy cannot arrive at the more complete result `(x < 1) & (y > 0)`.
```

Comment:
The solution given by `reduce_inequalities` is complete.

Reply:
I was using the language from @asmeurer

> (0 < y) & (y < oo) & (x > 1 - y) implies that y is in (0, oo), so x > 1 - y would mean x is in (-oo, 1), so a more complete solution would be (x < 1) & (y > 0).

Is there a better way to express this limitation (if you agree it is one) of `reduce_inequalities`?

Reply:
The result `(x < 1) & (y > 0)` is not implied by the original inequalities. As mentioned by @rdorsch above `x = 5` and `y = 5` would satisfy the original inequalities but clearly would not satisfy `x < 1`.

Reply:
Removed that note about "complete result."